app/api/handler/account/cookie.go

```diff
@@ -33,6 +33,9 @@ func includeTokenCookie(
 		cookie.Expires = time.UnixMilli(*tokenResponse.Token.ExpiresAt)
 	}
 
+	// Set the Secure attribute to true
+	cookie.Secure = true
```

Comment:
A cookie with the Secure attribute is only sent to the server with an encrypted request over the HTTPS protocol.
[MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies)
why we enforce true here?, bcoz in newEmptyTokenCookie there is already isSecure value.